sdk/test-utils/recorder/package.json

```diff
@@ -68,7 +68,7 @@
     "@azure/core-http": "^2.0.0",
     "@azure/core-rest-pipeline": "^1.1.0",
     "@azure/core-auth": "^1.3.2",
-    "@azure/core-util": "^1.0.0",
+    "@azure/core-util": "^1.0.1",
```

Comment:
Is this change necessary?

Reply:
I think it may be because core-util 1.0.0 doesn't have the new functions so code using it will break if 1.0.0 is installed which meets the ^

Reply:
Yeah but I don't see the recorder using any of the new stuff.

Reply:
I added a few unnecessary dependency bumps and missed this one when I reverted them. Fixed!